cppser/watchdog.cpp

```diff
@@ -113,11 +110,16 @@ namespace Spirit {
                     last_proc = lesson.endtime;
                 } catch (const nlohmann::json::parse_error& ex) {
                     log << "Wrong format from server: " << ex.what() << '\n';
+                } catch (const SQLError& ex) {
+                    log << "SQL Error: " << ex.what() << '\n';
                 }
             }
         } catch (const ErrorOpeningDatabase& ex) {
             log << "ErrorOpeningDatabase: " << ex.what() << '\n'
                 << "Exiting because of failure.\n";
+        } catch (const std::exception& ex) {
```

Comment:
If the database specified in `man.json` doesn't exist, the sqlite3 library will create an empty one for us. We need to add a few validations in program startup.